scripts/build/TestFx.targets

```diff
@@ -35,20 +35,21 @@
   <!-- Generate AssemblyInfo.cs -->
   <PropertyGroup>
     <TFBuildNumber Condition=" '$(TFBuildNumber)' == '' ">0.1</TFBuildNumber>
-    <MajorVersion>14.0</MajorVersion>
+    <LegacyMajorVersion>14</LegacyMajorVersion>
+    <LegacyMinorVersion>0</LegacyMinorVersion>
```

Comment:
It seems we are passing the version from the command line, is there any reason to keep these?

Reply:
Yes, these are defaults for when you build it from VS.

Reply:
Ah right! Let me add a comment and do the same for the other properties I have added.